<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QWidget" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	min-width: 50;
	min-height: 50;
	max-width: 50;
	max-height: 50;
}

QWidget#bottombar {
    background-color:  #f2f2f2;
	/*border-top: 3px solid qlineargradient(spread:pad, x1:0.539, y1:0.671, x2:0.521955, y2:0.113, stop:0.00568182 rgba(149, 149, 149, 150), stop:0.801136 rgba(210, 210, 210, 150));*/
	border-top: 3px solid qlineargradient(spread:pad, x1:0.539, y1:0.671, x2:0.521955, y2:0.113, stop:0.00568182 rgba(189, 189, 189, 150), stop:0.801136 rgba(220, 220, 220, 150));
}

QWidget#centralWidget {
   	/*background-color:  #f2f2f2;*/
	border:0px solid transparent;
	background-color: white;
}

QFrame#topbar {
	background-color: #1578ba;
	border: 0px solid transparent;
}

QPushButton#button_home 
{
	border-image: url(:/res/icons/ic_home_unchecked.png);
}

QPushButton#button_home:checked
{
	border-image: url(:/res/icons/ic_home_unchecked.png);
}

/*QFrame#bottombar {
	background-image: url(:/res/icons/ic_bottombar_bk.png);
}*/

/*Line#line {
	background-color: rgb(239, 41, 41);
	width: 4px;
}*/
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="7,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="bottombar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>62</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="55,50,110,50,110,50,55">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>19</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_image">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>52</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>52</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>265</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_home">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>265</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_video">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>52</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>52</width>
             <height>52</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>131</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
